atm.js

```diff
@@ -1,3 +1,88 @@
+var checkingInput;
+var checkingBalance ;
+var currentBalance;
 $(document).ready(function(){
-// Uh oh -- it's saying `$` is undefined! Something's missing from `index.html`...
+  checkingInput = $("#checking > .input");
+  checkingBalance = $("#checking > .balance");
+  currentBalance = $("checkingBalance" + "checkingInput")
+  console.log("hello");
+  $(".deposit:first").on("click", function(evt){
+    evt.preventDefault();
+    var currentChecking = parseFloat($('.balance:eq(0)').html().split('$')[1])
+
+    $(".balance:eq(0)").html("$" + (parseFloat(checkingInput.val())+currentChecking).toFixed(2));
+    currentChecking = parseFloat($('.balance:eq(0)').html().split('$')[1])
```

Comment:
i was surprised to have just learned that `:eq(x)` functions the same way as `.eq(x)` does. 👍